pkg/operation/botanist/component/istio/charts/istio/istio-istiod/templates/deployment.yaml

```
@@ -18,7 +18,7 @@ spec:
   template:
     metadata:
       labels:
-{{ .Values.labels | toYaml | indent 8 }}
+{{ .Values.labels | toYaml | indent 8 }}networking.gardener.cloud/to-seed-apiserver: allowed
```

It looks a bit strange if everything is on the same line. Maybe move the new label before the template.

```suggestion
        networking.gardener.cloud/to-seed-apiserver: allowed
{{ .Values.labels | toYaml | indent 8 }}
```

---

Thanks for changing it.